From adae9bf218efc9e43f24bda6989f09aa63141694 Mon Sep 17 00:00:00 2001
From: Konstantin Kostiuk <kkostiuk@redhat.com>
Date: Mon, 16 Dec 2024 17:45:52 +0200
Subject: [PATCH 4/4] qga: Add log to guest-fsfreeze-thaw command
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: 6-dehan <demeng@redhat.com>
RH-MergeRequest: 324: qga: Add log to guest-fsfreeze-thaw command
RH-Jira: RHEL-56340
RH-Acked-by: Konstantin Kostiuk <None>
RH-Acked-by: Yan Vugenfirer <None>
RH-Commit: [1/1] 794ed8728c0ecc0c9050c6d5380cc2f8d7bb7723 (6-dehan/centos-qemu-kvm)

Signed-off-by: Konstantin Kostiuk <kkostiuk@redhat.com>
Reviewed-by: Daniel P. Berrangé <berrange@redhat.com>
---
 qga/commands-posix.c | 2 ++
 qga/commands-win32.c | 3 +++
 2 files changed, 5 insertions(+)

diff --git a/qga/commands-posix.c b/qga/commands-posix.c
index c2bd0b4316..49e40f9127 100644
--- a/qga/commands-posix.c
+++ b/qga/commands-posix.c
@@ -806,8 +806,10 @@ int64_t qmp_guest_fsfreeze_thaw(Error **errp)
     int ret;
 
     ret = qmp_guest_fsfreeze_do_thaw(errp);
+
     if (ret >= 0) {
         ga_unset_frozen(ga_state);
+        slog("guest-fsthaw called");
         execute_fsfreeze_hook(FSFREEZE_HOOK_THAW, errp);
     } else {
         ret = 0;
diff --git a/qga/commands-win32.c b/qga/commands-win32.c
index 61b36da469..1aea6cd167 100644
--- a/qga/commands-win32.c
+++ b/qga/commands-win32.c
@@ -1273,6 +1273,9 @@ int64_t qmp_guest_fsfreeze_thaw(Error **errp)
     qga_vss_fsfreeze(&i, false, NULL, errp);
 
     ga_unset_frozen(ga_state);
+
+    slog("guest-fsthaw called");
+
     return i;
 }
 
-- 
2.39.3

